crystal/distribution.yaml

```diff
@@ -1584,6 +1584,22 @@ repositories:
       url: https://github.com/ros-visualization/rqt.git
       version: crystal-devel
     status: maintained
+  rqt_action:
+    doc:
+      type: git
+      url: https://github.com/ros-visualization/rqt_action.git
+      version: crystal-devel
+    release:
+      tags:
+        release: release/crystal/{package}/{version}
+      url: https://github.com/ros2-gbp/rqt_action-release.git
+      version: 1.0.1-0
+    source:
+      test_pull_requests: true
```

Comment:
The repository does neither contain anything to compile nor tests so a PR job seems unnecessary.